README.md

```diff
@@ -20,11 +20,14 @@ gem "lita-google"
 
 * `safe_search` (String, Symbol) - The safe search setting to use. Possible values are `:active`, `:moderate`, and `:off`. Default: `:active`.
 
+* `excluded_domains` (Array) - Domains from which you never want to see results. It should be an array of domain strings.
+
 ### Example
 
 ```
 Lita.configure do |config|
   config.handlers.google.safe_search = :off
+  config.excluded_domains = ['gawker', 'funnyjunk', 'dailymail']
```

Comment:
Use double quoted strings for consistency. Let's make the TLD(s) required too. `["gawker.com", "funnyjunk.com", "dailymail.co.uk"]`